src/routes/graph/find-edge.ts

```diff
@@ -35,8 +37,12 @@ interface Response {
 const controllerGeneratorOptions: ControllerGeneratorOptionsWithClient = {
   method: 'get',
   path: '/edge',
-  query: tableQuerySchemaGenerator(Joi.string().valid('hashId', 'name').default('hashId'))
+  query: tableQuerySchemaGenerator(Joi.string().valid('hashId', 'name', 'proximity').default('hashId'))
     .keys({
+      location: Joi.object().keys({
```

Comment:
I think this is a nice way to create the request. I think I might prefer it slightly over the current approach. However it is not consistent and I also do value consistency. Since it is unrealistic to change all locations where we already have coordinates for a slight preference I hope you would be willing to turn this implemenation into something closer to update-location.ts. The limits (Joi.number().min(-180).max(180).required(),) will be an improvement and the unnamed coordinates not so much but at least then we will be consistent with ourselves.